CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## [1.2.7](https://github.com/ably/ably-flutter/tree/v1.2.7)
+
+[Full Changelog](https://github.com/ably/ably-flutter/compare/v1.2.6...v1.2.7)
+
+**Implemented enhancements:**
+
+- Replace optional fields with default values [\#230](https://github.com/ably/ably-flutter/issues/230)
+- Refactor example app to be more readable [\#212](https://github.com/ably/ably-flutter/issues/212)
+- AuthCallback and InstanceStore Refactoring and documentation [\#250](https://github.com/ably/ably-flutter/pull/250) ([ben-xD](https://github.com/ben-xD))
+- Serialize LogLevel using constants [\#244](https://github.com/ably/ably-flutter/pull/244) ([ben-xD](https://github.com/ben-xD))
+- Replace deprecated method calls in tests [\#243](https://github.com/ably/ably-flutter/pull/243) ([ben-xD](https://github.com/ben-xD))
+- Example app: Paginated results viewer widget [\#242](https://github.com/ably/ably-flutter/pull/242) ([ben-xD](https://github.com/ben-xD))
+- Example app: Move out Realtime and Rest UI to separate widget and service [\#240](https://github.com/ably/ably-flutter/pull/240) ([ben-xD](https://github.com/ben-xD))
+- Example app: Extract System Details sliver into new widget [\#239](https://github.com/ably/ably-flutter/pull/239) ([ben-xD](https://github.com/ben-xD))
+- Hide API key secret inside API key UI [\#237](https://github.com/ably/ably-flutter/pull/237) ([ben-xD](https://github.com/ben-xD))
+- Rename `pushSetOnBackgroundMessage` to `pushBackgroundFlutterApplicationReadyOnAndroid` [\#236](https://github.com/ably/ably-flutter/pull/236) ([ben-xD](https://github.com/ben-xD))
+- Enhancement/run unit tests in ide [\#234](https://github.com/ably/ably-flutter/pull/234) ([ben-xD](https://github.com/ben-xD))
```

Comment:
I'd err on the side of keeping stuff like this out of the changelog. Maybe the same for #212 as well.

Reply:
[212](https://github.com/ably/ably-flutter/issues/212) is a refactoring of the example app, which was done for the user's experience, I think it's good to keep that one.

I've removed #234 as users are probably not intending to run unit tests at all. 689e43076d3b04f5dd532628dd68e78920d75218